authors/apps/articles/views.py

```diff
@@ -514,58 +514,65 @@ def post(self, request, slug):
         """
         Method that posts users article ratings
         """
-        rating = request.data.get("rate", {})
+        ratingData = request.data.get("rate", {})
 
-        rating_1 = rating.get('rating', None)
-
-        if not isinstance(rating_1, int):
+        rating = ratingData.get('rating', None)
+        note = ratingData.get('note', None)
+      
+        if not isinstance(rating, int):
             return Response(
                 {'message': 'Rating should be an integer'},
                 status=status.HTTP_401_UNAUTHORIZED)
-        if rating_1 < 1 or rating_1 > 5:
+        if rating <1 or rating > 5:
             return Response(
                 {'error': 'Rating should be between 1 and 5'},
                 status=status.HTTP_401_UNAUTHORIZED)
-
-        serializer = self.serializer_class(data=rating)
-        serializer.is_valid(raise_exception=True)
-        rating = serializer.data.get('rating')
+        
+        # serializer = self.serializer_class(data=rating)
+        # serializer.is_valid(raise_exception=True)
+        # rating = serializer.data.get('rating')
         try:
             article = Article.objects.get(slug=slug)
         except Article.DoesNotExist:
             raise NotFound("An article with this slug does not exist")
 
-        current_ratings = ArticleRatings.objects.filter(rater=request.user,
-                                                        article=article).first()
-        article_rating = Article.objects.filter(slug=slug).first()
+        # current_ratings = ArticleRating.objects.filter(rater=request.user,
+        #                                  article=article).first()
+        # article_rating = Article.objects.filter(slug=slug).first()
 
```

Comment:
same as above?

Reply:
Noted, let me remove it. True, it is not necessary.